### PR TITLE
Docs update about MSI auth support on ADO.NET and OLEDB connection managers

### PR DESCRIPTION
**Pull request by lewischeng-ms**

### CODE REVIEW

**Comment by PRMerger6** [comment]
@lewischeng-ms : Thanks for your contribution! The author, @douglaslMS, has been notified to review your proposed change.

**Comment by lewischeng-ms** [comment]
@swinarko Please help review. Thanks.

File: docs/integration-services/connection-manager/ado-net-connection-manager.md [hunk 1]
```diff
@@ -77,6 +77,43 @@ manager: craigg
...
+    CREATE USER [your AAD group name] FROM EXTERNAL PROVIDER;
+    ```
+
+1. **Grant the Azure AD group needed permissions** as you normally do for SQL users and others. For example, run the following code:
+
+    ```sql
+    ALTER ROLE [role name] ADD MEMBER [your AAD group name];
+    ```
+
+1. **Configure managed identity authentication** for the ADO.NET connection manager. Set the connection manager property **ConnectUsingManagedIdentity** to **True**. Please note that all other authentication methods (e.g., integrated authentication, password) configured on the ADO.NET connection manager will be **overrided** when managed identity authentication is used to establish database connection.
```
Comment (timchenyx) [inline docs/integration-services/connection-manager/ado-net-connection-manager.md]: Add how to configure it from the UI.

Comment (timchenyx) [inline docs/integration-services/connection-manager/ado-net-connection-manager.md]: Is there an UI element to enable this directly? If yes we should doc it.

File: docs/integration-services/connection-manager/ado-net-connection-manager.md [hunk 2]
```diff
@@ -77,6 +77,43 @@ manager: craigg
  **Delete**  
  Select a connection, and then delete it by using the **Delete** button.  
   
+### Managed Identities for Azure Resources Authentication
+When running SSIS packages in Azure-SSIS integration runtime in Azure Data Factory, you can use the [managed identity](https://docs.microsoft.com/en-us/azure/data-factory/connector-azure-sql-database#managed-identity) that is associated with your data factory for Azure SQL Database authentication. The designated factory can access and copy data from or to your database by using this identity.
+
+To use managed identity authentication, follow these steps:
```
Comment (timchenyx) [inline docs/integration-services/connection-manager/ado-net-connection-manager.md]: Are the steps same for both Azure SQL Database and MI?

File: docs/integration-services/connection-manager/ole-db-connection-manager.md [hunk 3]
```diff
@@ -81,6 +81,49 @@ manager: craigg
...
+1. **Grant the Azure AD group needed permissions** as you normally do for SQL users and others. For example, run the following code:
+
+    ```sql
+    ALTER ROLE [role name] ADD MEMBER [your AAD group name];
+    ```
+
+1. **Configure managed identity authentication** for the OLE DB connection manager.
+
+   1. Set the connection manager property **ConnectUsingManagedIdentity** to **True**. Please note that all other authentication methods (e.g., integrated security, password) configured on the OLE DB connection manager will be **overrided** when managed identity authentication is used to establish database connection.
+   1. In the connection manager property **ConnectionString**, set the connection property **Provider** to **MSOLEDBSQL**, which refers to the latest [**Microsoft OLE DB Driver for SQL Server**](https://go.microsoft.com/fwlink/?linkid=871294). To use managed identity authentication with OLE DB connection manager, this new provider must be used in your OLE DB connection manager.
```
Comment (timchenyx) [inline docs/integration-services/connection-manager/ole-db-connection-manager.md]: There is also the UI way to choose the new provider in addition to setting the connection string directly.

**Comment by lewischeng-ms** [comment]
I will send another PR to sql-docs-pr as internal contributor. So close this PR.

**Pull request closed without merging**